kos-core/source/kos/core/events/DefaultAsyncEventReplier.java

```diff
@@ -22,7 +25,7 @@ public void handle(AsyncResult<Void> result) {
         if (result.succeeded()) {
             message.reply(message.address());
         } else {
-            message.fail(1, result.cause().getMessage());
+            message.fail(1, result.cause().getMessage() + "\n" + ThrowableUtil.stackTraceToString(result.cause()));
```

Comment:
It might be worth manually rewrite this. It doesn't seems to be too complex.

```suggestion
            message.fail(1, result.cause().getMessage() + "\n" + stackTraceToString(result.cause()));
```

I'm dropping an example taken from another project. Feel free to adapt it to your suggested needs.
```java
    String stackTraceToString(Throwable throwable) {
        List<String> buffer = new ArrayList<>();
        traverse(throwable, buffer);
        return buffer.stream().collect(Collectors.joining("\n"));
    }

    private void traverse(Throwable throwable, List<String> buffer) {
        buffer.add(throwable.getClass().getSimpleName() + ": " + throwable.getMessage());

        for (StackTraceElement stackTraceElement : throwable.getStackTrace()) {
            buffer.add("\tat " + stackTraceElement);
        }

        if (throwable.getCause() != null) {
            traverse(throwable.getCause(), buffer);
        }
    }
```

Reply:
Yeah fair

Reply:
Cool updated to not use netty